log/operator

```diff
@@ -154,3 +154,5 @@ Caller sought to make a phone call.  I explained that I could not make a phone c
 Caller wanted me to call Vancouver 311, said she got the Portland Mayor. I instructed her to get a dialtone. Caller called back and said the call didn't work, I realized that a regional number would't work and told her. Caller told me she was trying to report an abandoned car in Vancouver, I looked up and told her the Vancouver parking enforcement number.
 2016-09-01 11:54 unknown Karl
 Caller told me she was trying to report an abandoned car in Vancouver, I looked up and told her the Vancouver parking enforcement number. Caller asked me how we were funded, I told her through grants. Caller thanked us for our work. Note, tihs call did not come through my cell, only my Futel phone, so I did not see the caller ID.
+2016-09-12  11:08 Alma
```

Comment:
format is date time location name
location is known from the incoming number
locations for numbers are found here, um, sorry
https://github.com/kra/futel-installation/blob/master/asteriskserver/src/etc/asterisk/extensions.conf#L16